src/components/Firestore/PanelHeader.scss

```diff
@@ -19,6 +19,11 @@
     padding: 8px 12px;
     height: 44px;
   }
+
+  .CardActionBar-actions {
+    // Icon buttons are padded, so we remove the double padding on the right.
+    margin-right: -12px;
```

Comment:
Why use margins to override padding (instead of directly setting padding)?

Reply:
So that action bars with no buttons won't go all the way to the right. Think about a panel header with a lot of text but no buttons on the right